app/assets/javascripts/oxalis/controller/combinations/skeletontracing_plane_controller.js

```diff
@@ -64,28 +62,18 @@ class SkeletonTracingPlaneController extends PlaneControllerClass {
     _.defer(() => this.simulateTracing(nodesPerTree, nodesAlreadySet + 1));
   }
 
-  start(): void {
-    super.start();
-    $(".skeleton-plane-controls").show();
```

Comment:
Just to be sure, this is no longer needed as the react tracing view takes care of it right?

Reply:
A blast from the past. :-D The class selector is long dead and was left behind to hunt us poor devs. With the reactification of the tracing view the correct controls are shown.
![giphy-downsized 1](https://user-images.githubusercontent.com/1105056/33214005-50377e42-d12a-11e7-952f-77b90a191f70.gif)